stripe/src/main/java/com/stripe/android/checkout/CheckoutViewModel.kt

```diff
@@ -36,13 +40,35 @@ internal class CheckoutViewModel internal constructor(
         mutableTransition.postValue(target)
     }
 
-    fun getPaymentMethods(
-        customerId: String,
+    /**
+     * Fetch the customer's saved payment methods.
+     * The activity args are assumed to be static across the lifetime of the view model
+     * so calling this method multiple times will not result in re-fetching the payment methods.
+     *
+     * If you wish to force an update, call [updatePaymentMethods] directly.
+     */
+    fun getPaymentMethods(activity: Activity): LiveData<List<PaymentMethod>> {
```

Comment:
Can we pass the `Intent` instance directly instead of passing an `Activity`?

Reply:
Sure, that's probably simpler